middleware/2024-10-10-1.3.64.md

```diff
@@ -0,0 +1,34 @@
+---
+authors: poscreator
+slug: middleware/1.3.64
+tags: [Middleware, Germany]
+---
+
+# Middleware 1.3.64 (Germany)
+In this version of the Middleware, we've added the migration receipt for switching local installations to the CloudCashbox, and fixed an issue that prevented DSFinV-K exports on Queues that were initially set-up during the pilot phase in Germany.
+
+<!--truncate-->
+
+:::caution
+
+Version 1.3 of the Middleware is meant for the German and Italian markets only, customers in Austria and France should continue to use version 1.2. We are currently testing a unifyed version for all markets, and will release an update about this soon.

```

Comment:
*unified

Reply:
Great that nobody noticed this for 3 years :D

Reply:
Fixed, thanks